DSCResources/MSFT_xRunbookDirectory/MSFT_xRunbookDirectory.schema.mof

```diff
@@ -0,0 +1,10 @@
+[ClassVersion("1.0.0.0"), FriendlyName("RunbookDirectory")]
+class MSFT_xRunbookDirectory : OMI_BaseResource
+{
+    [Key, Description("Path to Runbook(s) to be imported. Accepts wildcards.")] String RunbookPath;
+    [Write, Description("Describes the published state of the imported Runbook(s). Defaults to true, publishing the Runbook.")] Boolean Publish;
```

Comment:
For runbooks, this could be 3 states if we wanted to apply the Ensure pattern; Absent, Draft, Published. Does having 3 states violate the pattern?